### ISSUE TITLE
[Feature Request] tfjs-vis fitCallbacks should allow plotting multiple curves in the same axes

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#812]
Plotting multiple curves in the same axes will be very useful. In fact, it is a little inconvenient if this feature isn't there. For example, often we want to plot the training loss and the validation loss in the same axes in order to compare them, so we can make judgments about overfitting. 

In terms of API, this can be done in a fully backward-compatible way, by changing the signature of `metrics` from `string[]` to `Array<string | string[]>`. If an item is an array of `string`s, then it'll create an axes with multiple curves inside.
